src/Mod/Draft/draftmake/make_wire.py

```diff
@@ -68,24 +68,34 @@ def make_wire(pointslist, closed=False, placement=None, face=None, support=None,
     """
     if not App.ActiveDocument:
         App.Console.PrintError("No active document. Aborting\n")
-        return
+        return None
 
     import Part
 
-    if not isinstance(pointslist, list):
-        e = pointslist.Wires[0].Edges
-        pointslist = Part.Wire(Part.__sortEdges__(e))
-        nlist = []
-        for v in pointslist.Vertexes:
-            nlist.append(v.Point)
-        if DraftGeomUtils.isReallyClosed(pointslist):
-            closed = True
-        pointslist = nlist
+    if isinstance(pointslist, (list,tuple)):
+        for pnt in pointslist:
+            if not isinstance(pnt, App.Vector):
+                App.Console.PrintError(
+                    "Items must be Base.Vector objects\n".format(
+                    type(pointslist)))
```

Comment:
The expression inside format() is not used in the string.
Did you mean something like :
`App.Console.PrintError("Items must be Base.Vector objects, not {}.\n".format(type(pnt)))`

Reply:
Thanks @tomate44. I force push the change.